content/en/blog/_posts/2021-04-30-using-finalizers-to-control-deletion.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: 'Using Finalizers to Control Deletion'
+date: 2021-04-30
+slug: using-finalizers-to-control-deletion
+---
+
+**Authors:** Aaron Alpar (Kasten)
+
+Deleting objects in Kubernetes can be challenging. You may think you’ve deleted something, only to find it still persists. While issuing a `kubectl delete` command and hoping for the best might work for day-to-day operations, understanding how Kubernetes `delete` commands operate will help you understand why some objects linger after deletion. 
+
+In this post, I’ll look at:
+
+- What properties of a resource govern deletion
+- How finalizers and owner references impact object deletion
+- How the propagation policy can be used to change the order of deletions
+- How deletion works, with examples
+
+For simplicity, all examples will use ConfigMaps and basic shell commands to demonstrate the process. We’ll explore how the commands work and discuss repercussions and results from using them in practice.
+
+## The basic `delete`
+
+Kubernetes has several different commands you can use that allow you to create, read, update, and delete objects. For the purpose of this blog post, we’ll focus on four `kubectl` commands: `create`, `get`, `patch`, and `delete`. 
+
+Here are examples of the basic `kubectl delete` command:
+
+```
+$ kubectl create configmap mymap
+configmap/mymap created

```

Comment:
minor nit: You could remove the shell prompt ($)

Reply:
I removed the command prompts.

Reply:
@kbhawkey done.